### PR TITLE
add examples for  GPTJ

### PR DESCRIPTION
**Pull request by changwangss**
Signed-off-by: Wang, Chang1 <chang1.wang@intel.com>

## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by kevinintel** [comment]
what's the perf?

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3521/

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3542/

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3556/

**Review by PenghuiCheng — APPROVED** [review]

**Comment by chensuyue** [comment]
Pls update in readme model list: https://github.com/intel/neural-compressor/blob/wangchang/gptj/examples/README.md

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+sentencepiece != 0.1.92
```
[the remark this answers is not in the document]
Comment (changwangss) [reply examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/requirements.txt:1]: append now.

**Review by chensuyue — APPROVED** [review]

**Pull request merged**